CHANGELOG.md

```diff
@@ -3,6 +3,26 @@ Datadog Agent 6 has been officially released and the release notes can be found
 Changes
 =======
 
+# 5.32.8 / Unreleased 
+
+**Linux, Windows, Docker and Source Install**
+
+### Changes
+
+* [BUGFIX] Do not always start the profiler in dev mode. See #3888.
+* [BUGFIX] Update deprecated `datadog-cert.pem` certificate. See [#3882][] (Thanks [@sodabrew][]).
+* [BUGFIX] JMXFetch: bump to 0.26.8. Fixes Java6 compatibility + misc fixes. See #3894 and [omnibus-software-354](https://github.com/DataDog/omnibus-software/pull/354).  
```

Comment:
I don't think we ever broke Java 6 compat in a released agent 5, so let's not mention this.

Since we were shipping JMXFetch `0.26.3` in 5.32.7 (see https://github.com/DataDog/omnibus-software/blob/5.32.7/config/software/jmxfetch.rb#L4), I think we can replace this line with something like:

```suggestion
* [BUGFIX] JMXFetch upgraded from `0.26.3` to `0.26.8`. Upgrades various dependencies. See [JMXFetch's changelog](https://github.com/DataDog/jmxfetch/blob/agent-v5/CHANGELOG.md#0268--2020-09-15).
```